gtk/Gir.toml

```diff
@@ -411,7 +411,6 @@ generate_builder = true
 name = "Gtk.Arrow"
 #deprecated since 3.14
 status = "ignore"
-ignore_builder = true
```

Comment:
can't the type just be moved to the generic `ignore = [ ]`?

Reply:
Good idea. Did a small cleanup.